applications/minotari_console_wallet/src/grpc/wallet_grpc_server.rs

```
@@ -704,7 +750,7 @@ impl wallet_server::Wallet for WalletGrpcServer {
         let mut transfers = Vec::new();
         for (hex_address, address, amount, fee_per_gram, payment_type, user_payment_id, raw_payment_id) in recipients {
             let payment_id = if !raw_payment_id.is_empty() {
-                PaymentId::from_bytes(&raw_payment_id)
+                PaymentId::open(raw_payment_id.to_vec(), TxType::PaymentToOther)
```

It would be better if this option were completely removed from the interface, as it boils down to being the same as `UserPaymentId::user_bytes`. Having a choice between two things that give the same result is confusing. It is also possible to specify both `raw_payment_id` and `user_payment_id` with undefined behaviour for the user, where `raw_payment_id` will always have preference.

---

This is backwards compatibility, by removing this, its a silent fail. 
If we remove the grpc as well, it becomes a silent fail as well.